### PR TITLE
force users to accept license terms

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Please include a summary of the change and which issue is fixed.

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

### CODE REVIEW

File: src/utils/MSFraggerAdapter.cpp [hunk 1]
```diff
@@ -189,7 +191,20 @@ class TOPPMSFraggerAdapter final :
...
+                                  "MS Fragger application developed by Michigan within the OpenMS Pipeline and make available\n"
+                                  "for use on related service offerings supported by the University of Tubingen and the Center for\n"
+                                  "Integrative Bioinformatics.\n"
+                                  "Per the license agreement the use of the pipeline and associated materials is for academic\n"
+                                  "research, non-commercial or educational purposes. Any commercial use inquiries\n"
+                                  "must be directed to the University of Michigan Technology Transfer Office at\n"
+                                  "techtransfer@umich.edu. All right title and interest in MS Fragger shall remain with the\n"
+                                  "University of Michigan.\n"
+                                  "\n"
+                                  "Please see the supplied license file License.txt for details.\n"
```
Comment (jpfeuffer) [inline src/utils/MSFraggerAdapter.cpp:206]: How are users gonna find this txt in the different distributions?

Comment (timosachsenberg) [reply src/utils/MSFraggerAdapter.cpp:206]: I think we need to do two things:
- We need to distribute the EULA as txt file
- Make sure that the user agreed during installation or execution

I could, of course, also include the EULA text into the wrapper, but I am not sure if this is
legally sufficient.

Comment (jpfeuffer) [reply src/utils/MSFraggerAdapter.cpp]: Can you create an issue for that for the release?

Comment (timosachsenberg) [reply src/utils/MSFraggerAdapter.cpp]: let's do that when we include MSFragger in the thirdparty

**Pull request merged**